docs/landing-zone-v2/README.md

```diff
@@ -18,12 +18,6 @@
     - [Summary](#summary)
     - [Initial Organization configuration](#initial-organization-configuration)
   - [Config Controller project and cluster](#config-controller-project-and-cluster)
-    - [Option 1 - Org level folder](#option-1---org-level-folder)
```

Comment:
I'm wondering why these are being removed from the table of content.

also, has anybody tried replacing the entire table of content with just "[[_TOC_]]" and let github render it.

Reply:
GitHub's built-in Outline works well, even better for navigation because it's pinned on the side as you click/scroll

![image](https://github.com/GoogleCloudPlatform/pubsec-declarative-toolkit/assets/94859304/af49f15f-955d-4cc3-945c-73a646d35a2a)